### DISCUSSION TITLE
Small int data type

### DISCUSSION THREAD

**Discussion by Nick-Mazuk** [supabase/supabase#839] [Questions] [answered]
Looking at the [Postgres docs](https://www.postgresql.org/docs/9.1/datatype-numeric.html), Postgres has a small integer datatype that's just 2 bytes long.

Is there a way to add this datatype in Supabase? Currently, I only see options for `Int 4` and `Int 8`.

<img width="607" alt="Screen Shot 2021-03-08 at 4 56 23 PM" src="https://user-images.githubusercontent.com/5659171/110401849-38d16f80-802f-11eb-83a1-4c79a8f15201.png">

**Comment by kiwicopple** [comment, marked as answer]
Good call! Creating an issue here: https://github.com/supabase/supabase/issues/856

**Reply by kiwicopple** [reply]
(FYI, for now the workaround is to use the SQL editor)

**Reply by Nick-Mazuk** [reply]
Thanks @kiwicopple !

**Reply by gacurl** [reply]
@kiwicopple - nvm. Cleaned it all up

**Reply by inian** [reply]
This feature is now in the dashboard
